Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -1,26 +1,65 @@
 ![MRTK](../Images/EyeTracking/mrtk_et_targetselect.png)
 
-# Eye-Supported Target Selection
+# Eye-supported target selection
 This page discusses different options for accessing eye gaze data and eye gaze specific events to select targets in MRTK. 
-Eye Tracking allows for fast and effortless target selections using a combination of information about what a user is looking at with additional inputs such as 
-_hand tracking_ and _voice commands_:
-    - Look & Pinch (i.e., hold up your hand in front of you and pinch your thumb and index finger together)
-    - Look & Say _"Select"_ (default voice command)
-    - Look & Say _"Explode"_ or _"Pop"_ (custom voice commands)
-    - Look & Bluetooth button
-
-## Target Selection
-### Use Generic Focus Handler 
-If Eye Tracking is set up correctly (see [Basic MRTK Setup to use Eye Tracking](EyeTracking_BasicSetup.md)), enabling users to select 
+Eye tracking allows for fast and effortless target selections using a combination of information about what a user is looking at with additional inputs such as _hand tracking_ and _voice commands_:
+- Look & Say _"Select"_ (default voice command)
+- Look & Say _"Explode"_ or _"Pop"_ (custom voice commands)
+- Look & Bluetooth button
+- Look & Pinch (i.e., hold up your hand in front of you and bring your thumb and index finger together)
+    - _Please note that for this to work, the [hand rays need to be disabled](EyeTracking_EyesAndHands.md)!_
+
+To select holographic content using eye gaze, there are several options:
+
+**1. Use the primary focus pointer:** 
+This can be understood as your prioritized cursor. 
+By default, if the hands are in view, then this would be hand rays.
+If no hands are in view, then the prioritized pointer would be head or eye gaze. 
+Thus, please note that based on the current design head or eye gaze is suppressed as a cursor input if hand rays are used.
+
+For example: 
+A user wants to select a distant holographic button.
+As a developer, you want to provide a flexible solution that allows the user to achieve this tasks in various conditions:
+- Walk up to the button and poke it
+- Look at it from a distance and say "select"
+- Target the button using a hand ray and performing a pinch
+In this case, the most flexible solution is to use the primary focus handler as it will notify you whenever the currently prioritized primary focus pointer triggers an event. 
+Please note that if hand rays are enabled, the head or eye gaze focus pointer are disabled as soon as the hands come into view.
+
+**IMPORTANT:** Please note that if hand rays are enabled, the head or eye gaze focus pointer are disabled as soon as the hands come into view.
+If you want to support a [_'look and pinch'_ interaction, you need to disable the hand ray](EyeTracking_EyesAndHands.md).
```

Comment:
Same comment as above on providing a direct link to the hand ray disable guidance section - don't just link to the general page - link to the specific place :)

Reply:
Did you click on the page? It consists of 1 title and 2 paragraphs 🤨

Reply:
Well, alright. I added more titles to the one-pager and added a link here: EyeTracking_EyesAndHands.md#how-to-disable-the-hand-ray